packages/hooks/src/useCollectionQuery/useCollectionQuery.test.tsx

```diff
@@ -378,7 +377,7 @@ describe("useCollectionQuery", () => {
     describe("#subscribeToMore", () => {
       describe("when hook receives update from item not in collection", () => {
         it("should add new item to collection", async () => {
-          const { result, waitForNextUpdate } = renderHook(
```

Comment:
I must have missed this somewhere, and it's not a blocker at all - I'm just curious and can/should look it up myself

did `waitForNextUpdate` get removed? what's the replacement